README.md

```diff
@@ -7,18 +7,18 @@ A simple, scalable, and powerful architecture for building production ready Reac
 
 ## Introduction
 
-React is a great tool for building frontend applications. It has a very diverse ecosystem with hundreds of great libraries for literally anything you might need. However, it can be overwhelming to be forced to make so many choices.
-It is also very flexible, you can write React applications in any way you like but that flexibility comes with a cost. Since there is no pre-defined architecture developers can follow, it often leads to messy, inconsistent, or over-complicated codebases.
+React is a great tool for building frontend applications. It has a very diverse ecosystem with hundreds of great libraries for literally anything you might need. However, being forced to make so many choices can be overwhelming.
+It is also very flexible, you can write React applications in any way you like, but that flexibility comes with a cost. Since there is no pre-defined architecture developers can follow, it often leads to messy, inconsistent, or over-complicated codebases.
 
-This is an attempt to present a way of creating React applications using the best tools in the ecosystem with a good project structure that scales very well. It is based on the experience of working with many different codebases, and this architecture turns out to be the most effective one.
+This attempts present a way of creating React applications using the best tools in the ecosystem with a good project structure that scales very well. It is based on the experience of working with many different codebases, and this architecture turns out to be the most effective one.
```

Comment:
Shouldn't it be "This attempts to present....."

Reply:
Yes , agreed it should be "This attempts to present....." , the only thing i wanted to to put here was every PR should be closed with the proper comments , Thanks 😄

Reply:
I absolutely agree, this was closed because there was a duplicate which I closed in favor of this one. And now I see you are not the author of the PR :)

Reply:
@freddy1020 Can you make this small change so we can merge this?
FYI I decided to close https://github.com/alan2207/bulletproof-react/pull/83 since this is better. Thanks!

Reply:
Yep ! I'll do this change .